faq.html

```diff
@@ -1,93 +1,81 @@
 ---
 title: FAQ
+longTitle: Frequently Asked Questions
 layout: page
-editlink:
+editLink:
   template: faq.html
   data: _data/faq.yml
 ---
 
 {% assign languages = "en de" | split: ' ' %}
 
-<script type="text/javascript">
-    var languages = [ "{{ languages | join: '", "' }}" ];
+<script>
+    const languages = ["{{ languages | join: '", "' }}"]
 
     function switchLanguage(lang) {
         if ($.inArray(userLang, languages) > -1) {
-            $('.faqitem div[lang], h2[lang], .faqnavigation[lang]').hide();
-            $('.faqitem div[lang="' + lang + '"], h2[lang="' + lang + '"], .faqnavigation[lang="' + lang + '"]').show();
+            $('.faq-item div[lang], .faq-category-header[lang], .faq-navigation[lang]').hide()
+            $(`.faq-item div[lang="${lang}"], .faq-category-header[lang="${lang}"], .faq-navigation[lang="${lang}"]`).show()
 
-            // jump to anchor again because page length might have changed
+            // Jump to anchor again because page length might have changed
             if (location.hash) {
-                location.hash = location.hash;
+                location.hash = location.hash
             }
         }
     }
 </script>
 
-<h1>FAQ - Frequently Asked Questions</h1>
+<h1>FAQ</h1>
 
-<div id="language-selection" style="display: none;">
-    <a href="javascript:switchLanguage('en');">English</a>
-    <a href="javascript:switchLanguage('de');">Deutsch</a>
-</div>
+<p>Welcome to our FAQ where we collect and answer frequently asked questions.
+To quickly find an answer to your question, use the <strong>Find in page</strong>
+feature of your browser - usually accessible by pressing <kbd>Ctrl+F</kbd>.</p>
+<p>If you still don't find the answer you're looking for, check out our Support page.</p>
```

Comment:
Which is the "Support page" (link it)? Will the user find additional information on that page? Maybe reword a little bit (something like "contact us").